lightning/src/onion_message/packet.rs

```diff
@@ -314,6 +314,16 @@ pub(crate) enum ControlTlvs {
 	Receive(ReceiveTlvs),
 }
 
+/// A wrapper struct that stores the largest packet length in the given [`BlindedPath`].
+/// This helps us calculate the appropriate padding size for the [`ControlTlvs`] at the time
+/// of writing them.
```

Comment:
nit: s/at the time of/when

Reply:
Done! [pr3177.06](https://github.com/shaavan/rust-lightning/commits/pr3177.06)
Thanks!